src/__test__/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/DiffExprResults.test.jsx

```diff
@@ -214,4 +216,18 @@ describe('DiffExprResults', () => {
       lookupComponent.unmount();
     });
   });
+  it('Show comparison settings button works.', () => {
```

Comment:
you should also add unit tests to check:
- if "show settings" is clicked, it changes to "hide settings" and also displays the selected menu options
- if "hide settings" is clicked, then the selected menu options get hidden and the button changes to "show settings"
-